test/hotspot/jtreg/compiler/vectorapi/VectorSaturatedOperationsTest.java

```diff
@@ -0,0 +1,474 @@
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.SADD_VS, " >0 "}, applyIfCPUFeature = {"avx", "true"})
+    @Warmup(value = 10000)
+    public void sadd_short() {
+        for (int i = 0; i < COUNT; i += sspec.length()) {
+            ShortVector.fromArray(sspec, short_in1, i)
+                     .lanewise(VectorOperators.SADD,
+                               ShortVector.fromArray(sspec, short_in2, i))
+                     .intoArray(short_out, i);
+        }
+    }
+
+    @Check(test = "sadd_short")
+    public void sadd_short_verify() {
+        for (int i = 0; i < COUNT; i++) {
+            short actual = short_out[i];
+            short expected = VectorMath.addSaturating(short_in1[i], short_in2[i]);
+            if (actual != expected) {
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.SADD_VI, " >0 "}, applyIfCPUFeature = {"avx", "true"})
+    @Warmup(value = 10000)
+    public void sadd_int() {
+        for (int i = 0; i < COUNT; i += ispec.length()) {
+            IntVector.fromArray(ispec, int_in1, i)
+                     .lanewise(VectorOperators.SADD,
+                               IntVector.fromArray(ispec, int_in2, i))
+                     .intoArray(int_out, i);
+        }
+    }
+
+    @Check(test = "sadd_int")
+    public void sadd_int_verify() {
+        for (int i = 0; i < COUNT; i++) {
+            int actual = int_out[i];
+            int expected = VectorMath.addSaturating(int_in1[i], int_in2[i]);
+            if (actual != expected) {
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.SADD_VL, " >0 "}, applyIfCPUFeature = {"avx", "true"})
+    @Warmup(value = 10000)
+    public void sadd_long() {
+        for (int i = 0; i < COUNT; i += lspec.length()) {
+            LongVector.fromArray(lspec, long_in1, i)
+                     .lanewise(VectorOperators.SADD,
+                               LongVector.fromArray(lspec, long_in2, i))
+                     .intoArray(long_out, i);
+        }
+    }
+
+    @Check(test = "sadd_long")
+    public void sadd_long_verify() {
+        for (int i = 0; i < COUNT; i++) {
+            long actual = long_out[i];
+            long expected = VectorMath.addSaturating(long_in1[i], long_in2[i]);
+            if (actual != expected) {
+                throw new AssertionError("Result Mismatch : actual (" +  actual + ") !=  expected (" + expected  + ")");
+            }
+        }
+    }
+
+    @Test
+    @IR(counts = {IRNode.SADD_VB, " >0 " , "unsigned_vector_node", " >0 "}, phase = {CompilePhase.BEFORE_MATCHING}, applyIfCPUFeature = {"avx", "true"})
```

Comment:
```suggestion
    @IR(counts = {IRNode.SADD_VB, " >0 " ,
                  "unsigned_vector_node", " >0 "},
        phase = {CompilePhase.BEFORE_MATCHING},
        applyIfCPUFeature = {"avx", "true"})
```
I find this generally more readable, than a long line.
What is the `unsigned_vector_node` from, i.e. what is the whole line it maches on?